test/functional/apps/kibana_overview/_solutions.ts

```diff
@@ -33,20 +33,29 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       );
     });
 
-    it('contains the appropriate solutions', async () => {
+    it('contains Security and Observability solutions', async () => {
       const solutionCards = await find.allByCssSelector('.kbnOverviewMore__item');
-      expect(solutionCards.length).to.be(2);
+      expect(solutionCards.length >= 2).to.be(true);
 
-      const observabilityImage = await solutionCards[0].findByCssSelector('img');
-      const observabilityImageUrl = await observabilityImage.getAttribute('src');
-      expect(observabilityImageUrl.includes('/solutions_observability.svg')).to.be(true);
-
-      const securityImage = await solutionCards[1].findByCssSelector('img');
-      const securityImageUrl = await securityImage.getAttribute('src');
-      expect(securityImageUrl.includes('/solutions_security_solution.svg')).to.be(true);
+      const imageSrcs = [];
+      const re = /.*(\/solutions_(observability|security_solution)\.svg)/;
+      const myRegexp = new RegExp(re, 'g');
+      for (let i = 0; i < solutionCards.length; i++) {
+        const solutionCard = solutionCards[i];
+        const image = await solutionCard.findByCssSelector('img');
+        const imageSrc = await image.getAttribute('src');
+        const match = myRegexp.exec(imageSrc);
+        myRegexp.lastIndex = 0;
+        if (match && match.length > 1) {
+          imageSrcs.push(match[1]);
+        }
+      }
+      expect(imageSrcs.includes('/solutions_observability.svg')).to.be(true);
+      expect(imageSrcs.includes('/solutions_security_solution.svg')).to.be(true);
     });
 
-    it('click on Observability card leads to Observability', async () => {
+    // skipped as on Cloud it's a different setup than locally
+    xit('click on Observability card leads to Observability', async () => {
```

Comment:
Why do we keep those skipped tests if we can't run them on CI?

Reply:
So we know we need to tackle them at one point in time, which may not be right away.

Reply:
I see. Should we create an issue for it then and add the link in this comment?

Reply:
https://github.com/elastic/kibana/issues/139270

Reply:
Cool thanks. Sorry I meant in the comment of the code base 😊 So we know it is being tracked

Reply:
Hope it's ok now

Reply:
Yeah perfect thanks. It is just to be consistent with how the operation team flags skipped tests when there are too many failure in CI.